tests/acceptance/features/bootstrap/GraphContext.php

```diff
@@ -551,6 +573,26 @@ public function theUserCreatesNewUser(string $user, TableNode $table): void {
 		$this->featureContext->setResponse($response);
 	}
 
+	/**
+	 * @Given /^the user "([^"]*)" has created a new user using GraphAPI with the following settings:$/
```

Comment:
```suggestion
	 * @Given /^the user "([^"]*)" has created a new user with the following settings:$/
```
for the given step concise info. is generally not necessary imo

Reply:
also check for duplication if we have steps to do such this already, just in case

Reply:
> for the given step concise info. is generally not necessary imo

We have `When` step,
`When the user "Alice" creates a new user using GraphAPI with the following settings:`
step for user creation and just trying to make it consistent with that.